qiita_db/metadata_template/test/test_base_metadata_template.py

```diff
@@ -67,6 +67,60 @@ def test_clean_validate_template(self):
         with self.assertRaises(IncompetentQiitaDeveloperError):
             MT._clean_validate_template(None, 1)
 
+    def test_identify_forbidden_words(self):
+        """_identify_forbidden_words_in_column_names returns forbidden words
```

Comment:
Sometime ago we decided to not add docstrings to tests, could you remove? Applies to all methods below ...

Reply:
Sure thing.